ports/urdfdom/vcpkg.json

```
@@ -1,9 +1,10 @@
 {
   "name": "urdfdom",
-  "version": "1.0.4",
-  "port-version": 5,
+  "version": "3.1.0",
   "description": "Provides core data structures and a simple XML parsers for populating the class data structures from an URDF file.",
   "homepage": "https://github.com/ros/urdfdom",
+  "license": "BSD-2-Clause",
```

https://github.com/ros/urdfdom/blob/master/LICENSE is

```suggestion
  "license": "BSD-3-Clause",
```

When adding a license, make sure to compare the actual license text against SPDX.org (for example: https://spdx.org/licenses/BSD-3-Clause.html)

---

@ras0219-msft maybe we can utilize this tool somehow: https://github.com/spdx/spdx-license-matcher